### DISCUSSION TITLE
Error in firefox when setting an atom value

### DISCUSSION THREAD

**Discussion by relentless-coder** [pmndrs/jotai#2230] [Q&A]
I'm migrating our current Video Chat module from React Context API to jotai. And I am getting a DOMException error in Firefox even though the app continues to work.

This happens as soon as a second participant joins the room, the app in Firefox throws this error.

I had to set Event Listener breakpoints to get this stack trace, otherwise there's no stack trace in the console and just the error message `Uncaught (in promise) DOMException: The object can not be found here.`.

I can confirm that it's only happening with the introduction of Jotai. I tested the existing code, and it's working as expected.

In the stack trace, it always points to a file `useRoom.js` in my code, but every time points to a random setAtom call, so I'm not able to pinpoint which specific atom might be causing this issue. All atoms are primitive, except a room atom that is initialized with an Event Emitter

```
const room = atomWithReset(new EventEmitter());
```

I am setting a value to room atom in the `useRoom.js` file. Rest of the setAtom calls are just primitive values.

Any help is much appreciated.

```
Uncaught (in promise) DOMException: The object can not be found here.
    React 19
    unsub react.mjs:74
    flushPending vanilla.mjs:477
    flushPending vanilla.mjs:477
    flushPending vanilla.mjs:467
    writeAtom vanilla.mjs:369
    setAtom react.mjs:90
    connect useRoom.js:38
    fulfilled src_containers_VideoChat_components_PreJoinScreens_FeedbackScreen_index_js-src_containers_Vid-188a23.chunk.js:3456
    promise callback*step src_containers_VideoChat_components_PreJoinScreens_FeedbackScreen_index_js-src_containers_Vid-188a23.chunk.js:3468
    __async src_containers_VideoChat_components_PreJoinScreens_FeedbackScreen_index_js-src_containers_Vid-188a23.chunk.js:3469
    __async src_containers_VideoChat_components_PreJoinScreens_FeedbackScreen_index_js-src_containers_Vid-188a23.chunk.js:3453
    connect useRoom.js:29
    handleJoin DeviceSelectionScreen.js:94
    React 38
    unsub react.mjs:74
    flushPending vanilla.mjs:477
    flushPending vanilla.mjs:477
    flushPending vanilla.mjs:467
    writeAtom vanilla.mjs:369
    setAtom react.mjs:90
    resetAtom utils.mjs:8
    connect useRoom.js:50
4 react-dom.development.js:5899
    enter event-loop.js:82
    _pauseAndRespond thread.js:957
    _pauseAndRespondEventBreakpoint thread.js:886
    _makeEventBreakpointEnterFrame thread.js:867
    React 19
    unsub react.mjs:74
    flushPending vanilla.mjs:477
    forEach self-hosted:4356
    flushPending vanilla.mjs:477
    forEach self-hosted:203
    flushPending vanilla.mjs:467
    writeAtom vanilla.mjs:369
    setAtom react.mjs:90
    connect useRoom.js:38
    InterpretGeneratorResume self-hosted:1469
    next self-hosted:1390
    fulfilled src_containers_VideoChat_components_PreJoinScreens_FeedbackScreen_index_js-src_containers_Vid-188a23.chunk.js:3456
```

**Comment by dai-shi** [comment]
Is it a Firefox only problem?

**Reply by relentless-coder** [reply]
Yes, only happens in Firefox.

**Comment by dai-shi** [comment]
Does it reproduce if you don't use `atomWithReset` and set new EventEmmiter instead of resetting?

**Reply by relentless-coder** [reply]
Yes, it still throws the error.

**Reply by dai-shi** [reply]
What about if you don't use `EventEmmitter` at all?

Can you create a minimal reproduction with https://csb.jotai.org that fails in Firefox and works in other browsers?
I hope someone can help with such a reproduction.
